PWGCF/Flow/Tasks/flowPbpbPikp.cxx

```diff
@@ -238,6 +266,24 @@ struct FlowPbpbPikp {
     return 0;
   }*/
 
+  template <typename TTrack, typename vector, char... chars>
+  void resurrectParticle(TTrack trackplus, TTrack trackminus, vector plusdaug, vector minusdaug, vector mom, double plusmass, double minusmass, const ConstStr<chars...>& hist)
+  {
+    for (auto const& [partplus, partminus] : o2::soa::combinations(o2::soa::CombinationsStrictlyUpperIndexPolicy(trackplus, trackminus))) {
+      if (getNsigmaPID(partplus) != 2)
+        continue;
+      if (getNsigmaPID(partminus) != 2)
+        continue;
+
+      plusdaug = ROOT::Math::PxPyPzMVector(partplus.px(), partplus.py(), partplus.pz(), plusmass);
+      minusdaug = ROOT::Math::PxPyPzMVector(partminus.px(), partminus.py(), partminus.pz(), minusmass);
+      mom = plusdaug + minusdaug;
+
+      histos.fill(hist, mom.M());
+    }
+    return;
+  }
```

Comment:
It seems this routine is not used any longer. Please, remove it in the next iteration